bin/license-checker-ci

```diff
@@ -79,6 +79,15 @@ function readConfig() {
 	acceptedNonSpdx = partitions[1].concat('Project-Owner');
 }
 
+function satisfiesNoThrow(first, second) {
+	try {
+		return satisfies(first, second);
+	} catch (err) {
```

Comment:
probably it should log an error for better troubleshooting

Reply:
If a package has an invalid license expression (e.g. the `first` param), it will get logged on 120. For example, here is the output from the test case that explicitly checks for this:

```>>> Package "bluebird@3.4.7" doesn't meet license requirements ((MIT OR Invalid-SPDX-ID)).
    Repository: https://github.com/petkaantonov/bluebird
    Path: /home/travis/build/Brightspace/d2l-license-checker/test/data/proj-bad-spdx-expression/node_modules/bluebird
```

The `second` param is just the list of allowed licenses OR'd together. The allowed licenses are specified here:
https://github.com/Brightspace/d2l-license-checker/blob/master/config/default_config.json#L2

I'm not sure if we need to log them.